govuk-public-api-rules/package.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "govuk-public-api-rules",
```

Should it be clear these are spectral rules?

---

Perhaps:

```suggestion
  "name": "spectral-ruleset-govuk-public-api",
```

(We'd also want to rename the directory name / any references)